### PR TITLE
Now w metadatas

### PR DESCRIPTION
**Pull request by greg-snowflake**
Code is tested in dev and records metadata successfully.

### CODE REVIEW

File: src/runners/alert_queries_runner.py [hunk 1]
```diff
@@ -157,12 +176,35 @@ def query_for_alerts(query_name: str):
...
+    metadata['RUN_END_TIME'] = str(metadata['RUN_END_TIME'])       # then convert to string for json serializing
+    metadata['RUN_DURATION'] = str(metadata['RUN_DURATION'])
+
+    statement = f'''
+        INSERT INTO {METADATA_TABLE}
+            (event_time, v) select '{metadata['RUN_START_TIME']}',
+            PARSE_JSON(column1) from values('{json.dumps(metadata)}')
+        '''
+    try:
+        log.info("Recording run metadata...")
```
Comment (andrey-snowflake) [inline src/runners/alert_queries_runner.py]: same as before — either period ending or "done." after it's done?

File: src/runners/violation_suppressions_runner.py [hunk 2]
```diff
@@ -24,17 +31,44 @@ def run_suppression(squelch_name):
...
+        RUN_METADATA['QUERY_HISTORY'].append(metadata)
         log.fatal("Suppression query {squelch_name} execution failed.", e)
+        pass
 
     print(f"Suppression query {squelch_name} executed")
+    log.metadata_fill(metadata, status='success', rows=ctx.cursor().rowcount)
+    RUN_METADATA['QUERY_HISTORY'].append(metadata)
+
+
+def record_metadata(ctx, metadata):
```
Comment (andrey-snowflake) [inline src/runners/violation_suppressions_runner.py]: are these identical in the 4 files? can we put them in a helper?

Comment (greg-snowflake) [reply src/runners/violation_suppressions_runner.py]: record_metadata is slightly different between suppressions and queries, so we can't put it in a helper quite yet. When we get more robust failure logging for suppressions, that becomes a reasonable refactor.

File: src/runners/alert_queries_runner.py (hunk 1 above)
Comment (greg-snowflake) [reply src/runners/alert_queries_runner.py]: fixed!

**Pull request merged**